src/assets/img/icons/ui/close-lg.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 42 42"><style>.st0{fill:none;stroke:#fff;stroke-miterlimit:10}</style><path class="st0" d="M0 0l42 42M0 42L42 0"/></svg>
```

Comment:
Once we decide on an icon to use, this need to be moved to the design assets repo.